tools/example-templates/additional-files/Attr1Attribute.cs

```diff
@@ -0,0 +1,5 @@
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+public class Attr1Attribute : Attribute
+{
+    public Attr1Attribute() {}
```

Comment:
I would expect the compiler to generate this constructor for us by default - I don't think we need it explicitly declared.

Reply:
Agreed; removed